tldw_Server_API/app/core/DB_Management/Media_DB_v2.py

```
@@ -6865,6 +6979,1092 @@ def search_claims(
+        return len(insert_rows)
+
+    def replace_data_table_contents(
+        self,
+        table_id: int,
+        *,
+        columns: List[Dict[str, Any]],
+        rows: List[Dict[str, Any]],
+    ) -> Tuple[int, int]:
+        """Replace data table columns and rows, returning counts inserted."""
+        if not columns:
+            raise InputError("columns are required")
```

Ruff (0.14.13)

Avoid specifying long messages outside the exception class

(TRY003)